### ISSUE TITLE
fresh installation: `AttributeError: module 'isoduration' has no attribute 'DurationParsingException'`

### ISSUE THREAD

**Issue by alexlenail** [napari/napari#6220]
```
~ » python -m pip install "napari[all]"
~ » napari 
Traceback (most recent call last):
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/_lazy.py", line 45, in __getattr__
    submod = import_module(
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/viewer.py", line 8, in <module>
    from napari.components.viewer_model import ViewerModel
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/components/__init__.py", line 19, in <module>
    from napari.components.layerlist import LayerList
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/components/layerlist.py", line 9, in <module>
    from napari.layers import Layer
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/layers/__init__.py", line 9, in <module>
    from napari.layers.base import Layer
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/layers/base/__init__.py", line 2, in <module>
    from napari.layers.base.base import Layer, no_op
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/layers/base/base.py", line 14, in <module>
    from npe2 import plugin_manager as pm
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/__init__.py", line 11, in <module>
    from ._dynamic_plugin import DynamicPlugin
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/_dynamic_plugin.py", line 19, in <module>
    from ._plugin_manager import PluginManager
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/_plugin_manager.py", line 31, in <module>
    from ._command_registry import CommandRegistry
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/_command_registry.py", line 9, in <module>
    from .manifest import utils
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/manifest/__init__.py", line 2, in <module>
    from .schema import ENTRY_POINT, PluginManifest
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/manifest/schema.py", line 20, in <module>
    from .contributions import ContributionPoints
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/manifest/contributions/__init__.py", line 2, in <module>
    from ._configuration import ConfigurationContribution, ConfigurationProperty
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/manifest/contributions/_configuration.py", line 5, in <module>
    from ._json_schema import (
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/npe2/manifest/contributions/_json_schema.py", line 12, in <module>
    from jsonschema.exceptions import ValidationError
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/jsonschema/__init__.py", line 13, in <module>
    from jsonschema._format import FormatChecker
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/jsonschema/_format.py", line 499, in <module>
    raises=isoduration.DurationParsingException,
AttributeError: module 'isoduration' has no attribute 'DurationParsingException'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/alex/miniconda3/envs/py39/bin/napari", line 8, in <module>
    sys.exit(main())
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/__main__.py", line 564, in main
    _run()
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/__main__.py", line 213, in _run
    from napari import Viewer, run
  File "/Users/alex/miniconda3/envs/py39/lib/python3.9/site-packages/napari/_lazy.py", line 53, in __getattr__
    raise ImportError(
ImportError: Failed to import viewer from napari. See cause above
```
Hard to parse this error, not sure what imports are failing.

**Comment by Czaki** [comment]
Could you add the output of the `pip list` command?

**Comment by alexlenail** [comment]
I think my issue was unrelated to napari, sorry

**Comment by DavidGill159** [comment]
Hey @alexlenail What was your solution? I'm getting "AttributeError: module 'isoduration' has no attribute 'DurationParsingException'" in another context.

**Comment by alexlenail** [comment]
iirc think this was an issue with my python environment. I might have updated python or something, sorry to have forgotten.

**Comment by Czaki** [comment]
Try to reinstall `isoduration` package.
